src/rulesets/oas/index.json

```diff
@@ -340,7 +340,7 @@
       "message": "{{error}}",
       "recommended": true,
       "type": "validation",
-      "given": "$..[?(@.enum && @.type)]",
+      "given": "$..[?(@ && @.enum && @.type)]",
```

Comment:
seems like jsonpath-plus attempts to evaluate script filter expressions for falsy values.
If I remove `@` a TypeError is thrown due to some null value somewhere in the doc.